src/ProphecyTrait.php

```diff
@@ -49,6 +51,17 @@ protected function prophesize(?string $classOrInterface = null): ObjectProphecy
         return $this->getProphet()->prophesize($classOrInterface);
     }
 
+    protected function setUp(): void
```

Comment:
We should not override the `setUp` method to avoid issues with classes using the trait and not aliasing the trait method to call it as well. `@before` annotation should be used.

But here, the best solution IMO would be to register it in `prophesize`.